vmm/src/device_manager/legacy.rs

```diff
@@ -4,6 +4,8 @@
 // Portions Copyright 2017 The Chromium OS Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
 // found in the THIRD-PARTY file.
+#![cfg(target_arch = "x86_64")]
```

Comment:
I'm not completely sure what this means. Is this supposed to prevent this file from compiling on arm ?

If so wouldn't it be better to move this directive to `vmm/src/device_manager/mod.rs` before `pub mod legacy;` ?

Reply:
Yeah, that makes sense, will try it.